public/stylesheets/newApp.css

```
@@ -1,23 +1,62 @@
+#new-app {
+  padding-bottom: 15px;
+}
+
+.dropdown {
+  position: relative;
+}
+
+.dropdown-menu {
+  width: 63%;
+}
```

#new-app .dropdown-menu {
    width: 63%;
    will-change: transform;
    left: 50%;
    right: auto;
    text-align: center;
    transform: translate(-50%, 0);
}
```suggestion
.dropdown-menu {
  width: 63%;
}
```